requirements.txt

```
@@ -5,6 +5,7 @@ mkdocs-material==5.3.2
 mkdocs-monorepo-plugin~=0.4.16
 plantuml-markdown==3.4.2
 markdown_inline_graphviz_extension==1.1
+mdx_truly_sane_lists=1.2
```

I think the syntax is:

```suggestion
mdx_truly_sane_lists==1.2
```

---

thanks for the catch! added